File: -Errors-/P1168/P1168.py
test_cases = int(input())

for i in range(0, test_cases):

    summation = 0
    numbers = input()

    led = {
        "0": 6,
        "1": 2,
        "2": 5,
        "3": 5,
        "4": 4,
        "5": 5,
        "6": 6,
        "7": 3,
        "8": 7,
        "9": 6
    }

    for number in numbers:
        summation += led[number]

    print(f"{summation} leds")

File: Strings/P1168/src/P1168.kt
fun main(args: Array<String>) {

    val testCases = readLine()!!.toInt()

    for (i in 1..testCases) {

        var summation = 0
        val numbers = readLine()

        val ledNeeded = mutableMapOf(
            '0' to 6,
            '1' to 2,
            '2' to 5,
            '3' to 5,
            '4' to 4,
            '5' to 5,
            '6' to 6,
            '7' to 3,
            '8' to 7,
            '9' to 6
        )

        if (numbers != null) {
            for (number in numbers) {
                summation += ledNeeded[number]!!
            }
        }

        print("$summation leds\n")
    }
}